### DISCUSSION TITLE
Videos not downloaded cause DRM protection

### DISCUSSION THREAD

**Discussion by NolandTUX** [heliomarpm/udemy-downloader-gui#92] [Q&A] [answered]
Hello,
I am using Udeler 1.11.5 to download some udemy courses I have already purchased. The videos are not downloaded because there is a DRM protection. I have checked the "Keep downloading even if the course is DRM protected" but it does not work either.

Any idea to fix this?
Thanks a million

**Comment by sammirzagharcheh** [comment]
The same is here for me as well.

**Comment by heliomarpm** [comment, marked as answer]
Sorry guys, Udeler doesn't download protected videos

**Reply by Parth-Consul** [reply]
Is there any other way to download DRM protected files?

**Reply by heliomarpm** [reply]
> Is there any other way to download DRM protected files?

https://github.com/Puyodead1/udemy-downloader
